### ISSUE TITLE
Improve book list

### ISSUE THREAD

**Issue by kevinbowen777** [kevinbowen777/bookstore#15]
Remove bullets from list. Perhaps use the following css:

```
ul.no-bullets {
  list-style-type: none; /* Remove bullets */
  /* padding: 0;  Remove padding */
  margin: 0; /* Remove margins */
}
```

**Comment by kevinbowen777** [comment]
Setup book list and added from shell

book = Book.objects.create(
    title="Brave New World",
    author="Aldous Huxley",
    price="19.95")

book = Book.objects.create(
    title="Django for Beginners: Learn web development with Django",
    author="William S. Vincent",
    price="39.95",
    cover="covers/dfb_cover_40_4hqNCAw.jpg")

book = Book.objects.create(
    title="Django for APIs: Build web APIs with Python & Django",
    author="William S. Vincent",
    price="49.95",
    cover="covers/dfa_cover_40_krlHV6o.jpg")

book = Book.objects.create(
    title="Django for Professionals: Production websites with Python & Django",
    author="William S. Vincent",
    price="39.95",
    cover="covers/dfp_PXomXx9.png")

book = Book.objects.create(
    title="Atoms, Metaphors and Paradoxes: Niels Bohr and the Construction of a New Physics",
    author="Sandro Petruccioli",
    price="59.95",
    cover="covers/atoms_metaphors.jpg")

book = Book.objects.create(
    title="Luce elettricità magnetismo",
    author="Sandro Petruccioli",
    price="79.95",
    cover="covers/luce_petro.jpg")

book = Book.objects.create(
    title="The Structure of Scientific Revolutions",
    author="Thomas Kuhn",
    price="29.95",
    cover="covers/struct_sci_rev_kuhn.jpg")

book = Book.objects.create(
    title="Black-Body Theory and the Quantum Discontinuity, 1894-1912",
    author="Thomas Kuhn",
    price="49.95",
    cover="covers/black_body_kuhn.jpg")

book = Book.objects.create(
    title="Capitalist Realism: Is There No Alternative?",
    author="Mark Fisher",
    price="12.95",
    cover="covers/cap_fisher.jpg")

book = Book.objects.create(
    title="Ghosts of My Life: Writings on Depression, Hauntology and Lost Futures",
    author="Mark Fisher",
    price="26.95",
    cover="covers/ghosts_fisher.jpg")

book = Book.objects.create(
    title="The Conquest of Bread",
    author="Pyotr Kropotkin",
    price="9.95",
    cover="covers/bread_kropot.jpg")

book = Book.objects.create(
    title="Anarchism: A Collection of Revolutionary Writings",
    author="Pyotr Kropotkin",
    price="8.95",
    cover="covers/anarcho_kropot.jpg")

book = Book.objects.create(
    title="My Mother: Demonology",
    author="Kathy Acker",
    price="27.95",
    cover="covers/mymother_acker.jpg")

book = Book.objects.create(
    title="Blood and Guts in High School",
    author="Kathy Acker",
    price="23.95",
    cover="covers/blood_guts_acker.jpg")

book = Book.objects.create(
    title="Dubliners",
    author="James Joyce",
    price="9.95",
    cover="covers/dubliners_joyce.jpg")

book = Book.objects.create(
    title="Ulysses",
    author="James Joyce",
    price="16.95",
    cover="covers/uly_joyce.jpg")

Sandro Petruccioli
James Joyce
Thomas Kuhn
William S. Vincent
Aldous Huxley
Mark Fisher
Kathy Acker
Pyotr Kropotkin

**Comment by kevinbowen777** [comment]
Resolved with commit ad4c678
